src/ol/image.js

```diff
@@ -122,10 +122,11 @@ ol.Image.prototype.handleImageLoad_ = function() {
 
 
 /**
- * Load not yet loaded URI.
+ * Load not yet loaded or erroneous image.
```

Comment:
I think `Load the image or retry if loading previously failed.` would make for clearer docs.  Loading an "erroneous" image makes it sound like the intent is to load the *wrong* image.